elixir-sdk/native/sdk_core/Cargo.toml

```
@@ -10,7 +10,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 rustler = { version = "0.36.1", features = ["serde"] }
-eppo_core = { version = "=9.1.1", path = "../../../eppo_core", features = ["rustler"] }
+eppo_core = { version = "9.1.1", features = ["rustler"] }
```

I believe we still want to pin a specific version of core. Otherwise, it technically may ship unexpected changes for users who build from source
```suggestion
eppo_core = { version = "=9.1.1", features = ["rustler"] }
```

---

Fixed!